doc/manuals/admin/statistics.md

```diff
@@ -0,0 +1,215 @@
+## Statistics
+
+**Warning**: This section of the manual is yet work on progress, so maybe the current Orion implementation
+is not fully aligned with it. In addition, take into account that statistics API is by the momennt
+in beta status, so changes in the name of the different counters or the JSON structure could take
+place in the future.
+
+Orion Context Broker provides a set of statistics (mainly aimed at testing and debugging) through the
+`GET /statistics` and `GET /cache/statistics` operations. These operations only supports JSON encoding.
+
+At the present moment, statistics accuracy is not guaranteed under high local conditions. Atomic
+counters are not used in all places, so race conditions on request running at the same time could
+(theoretically) lead to losing some data. However, under low load conditions, statistics should
+be accurate. There is [an open issue about this](https://github.com/telefonicaid/fiware-orion/issues/1504).
+
+## GET /statistics
+
+The statistics JSON is composed of four conditional blocks and two unconditional fields:
+
+```
+{
+  "counters" : { ... },
+  "semWait" : { ... },
+  "timing" : { ... },
+  "notifQueue": { ... },
+  "uptime_in_secs" : "65697",
```

Comment:
Taking into account that we have removed the "XML burden" in statistics (*These operations only supports JSON encoding*) probably it makes sense to use JSON number instead of strings, eg:

```
"uptime_in_secs" : 65697,
```